multi-cloud/README.md

```diff
@@ -389,13 +402,32 @@ it out pretty easily on your own once this is done.
 
 IF you didn't monkey with the model:
 ```bash
+export EXAMPLE_SELDON=~/example-seldon
 cd $EXAMPLE_SELDON/workflows
 ~/argo submit training-sk-mnist-workflow.yaml -n kubeflow
 ```
 
-ELSE IF you monkeyed with the model, you'll need to train and build a new image:
+ELSE IF you monkeyed with the model, you'll need to train and build a new image.
```

Comment:
This is not correct. Please delete this whole block.

Reply:
This is actually required. You may already have the secret in your cluster, but other folks may not. Open question of it will automaticall grabs configured docker creds from the host though so I might try that quickly

Reply:
see https://github.com/kubeflow/example-seldon#train-the-models